packages/docusaurus-plugin-ideal-image/src/theme/IdealImage.js

```diff
@@ -14,14 +14,14 @@ function Image(props) {
       {...props}
       alt={alt}
       className={className}
-      height={img.src.height}
+      height={img.src.height || 100}
```

Comment:
in case someone wonders the hardcoded 100 value.

This is just to not throw error in dev because height and weight is a required field for react-ideal-image

see https://github.com/endiliey/responsive-loader#options
> Disable processing of images by this loader (useful in development). srcSet and other attributes will still be generated but only for the original size. Note that the width and height attributes will both be set to 100 but the image will retain its original dimensions.